src/components/Error/Error.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, { useContext, useEffect } from 'react';
+import { TodoContext } from '../../context';
+import { ErrorMessage } from '../../types/ErrorMessages';
+import { wait } from '../../utils/fetchClient';
+
+export const Error: React.FC = () => {
+  const { errorMessage, handleSetErrorMessage } = useContext(TodoContext);
+
+  useEffect(() => {
+    wait(3000).then(() => handleSetErrorMessage(ErrorMessage.None));
+  }, [errorMessage]);
```

Comment:
```suggestion
  }, [errorMessage, handleSetErrorMessage]);
```
you can add `handleSetErrorMessage` and remove: `/* eslint-disable react-hooks/exhaustive-deps */` I guess